netbox/templates/extras/objectchange.html

```diff
@@ -144,6 +145,14 @@ <h5 class="card-header">{% trans "Post-Change Data" %}</h5>
         </div>
     </div>
 </div>
+<div class="row mb-3">
```

Comment:
The `<div>` elements with class `row` should not have class `mb-3` applied as it results in extraneous vertical margin between cards. (It should have been removed from the existing elements during the v4.0 UI overhaul.)

Reply:
Would you like me to remove `mb-3` from all the others in this template?

Reply:
Yes, but I'll do it real quick.